Firmware/ultralcd.cpp

```diff
@@ -7783,8 +7800,7 @@ bool lcd_selftest()
 				_progress = lcd_selftest_screen(TestScreen::FsensorOk, _progress, 3, true, 2000); //fil sensor OK
 			}
 #endif //PAT9125
-//#ifdef IR_SENSOR_ANALOG
-#if (0)
+#ifdef IR_SENSOR_ANALOG
```

Comment:
The main trouble with this was that the user would have been forced to remove the loaded filament on an assembled printer - just to check the two voltages.
That's why there was the #if 0
Did you find a way around it?

Reply:
That is something I didn't know that assembled printers have filament loaded...

have to think about that

Reply:
This section is in the selftest and enables the check if the sensor is working correctly:
- If filament is loaded the user needs to unload and redo the selftest
- If filament is unloaded the user will need to load in the sensor (but not in extruder"

Reply:
yes, but that's why we moved the detection to runtime - not to force the user to remove and insert the filament